packages/@webex/plugin-cc/src/services/config/types.ts

```diff
@@ -61,50 +72,175 @@ export type AgentResponse = {
   /**
    * The email address of the agent.
    */
-
   email: string;
 
   /**
-   * This Specify the teams id which got assigned to the agent.
+   * Team IDs assigned to the agent.
    */
   teamIds: string[];
+
+  /**
+   * Skill profile ID of the agent.
+   */
+  skillProfileId: string;
+
+  /**
+   * Site ID of the agent.
+   */
+  siteId: string;
+
+  /**
+   * Database ID of the agent.
```

Comment:
question: What is the database ID of an agent and how is it used?

Reply:
We might not use this as per our current requirements,
But Agent desktop uses this so kept as it is.
Basically Its agentAnalyzerId/superviserId of the agent that Agentdesktop client  uses for getParticipantsDetails and  get supervisor name.